src/transformers/modeling_utils.py

```diff
@@ -3981,6 +3986,12 @@ def _fix_key(key):
                 return key.replace("gamma", "weight")
             return key
 
+        if "beta" in loaded_keys or "gamma" in loaded_keys:
+            logger.warning(
+                f"Parameter names `gamma` or `beta` for {cls.__name__} will be renamed within the model. "
+                f"Please use different names to suppress this warning."
+            )
```

Comment:
I don't this is quite right, this assumes the weight is called "beta" in the state dict, but it could be called "layer.beta"